lib/waterline/methods/find-one.js

```diff
@@ -317,10 +317,19 @@ module.exports = function findOne( /* criteria?, populates?, explicitCbMaybe?, m
               return proceed(undefined, thePopulatedRecord);
             }//-•
 
-            // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-            // FUTURE: This is where the `afterFindOne()` lifecycle callback would go
-            // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-            return proceed(undefined, thePopulatedRecord);
+            // If no afterCreate callback defined, just proceed.
```

Comment:
This comment & the one for `afterFind `are a bit misleading

Reply:
great catch, fixed